R/derive_var_ontrtfl.R

```diff
@@ -122,10 +122,10 @@
 #' )
 #'
 #' advs <- tibble::tribble(
-#'   ~USUBJID, ~ADTM,                ~TRTSDTM,                   ~TRTEDTM,                   ~TPT,
+#'   ~USUBJID, ~ADTM, ~TRTSDTM, ~TRTEDTM, ~TPT,
```

Comment:
Can we avoid removing the alignment?

Reply:
> Can we avoid removing the alignment?

Would we need to update STYLER checks? As I think adding indentation will lead to STYLER warnings.

Reply:
Just fixed this,